build.gradle

```diff
@@ -51,12 +51,11 @@ if (!hasProperty('StudioCompilePath')) {
 }
 
 intellij {
-    version 'IC-15.0.5' // this value is not important, as long as you can compile
-
     pluginName 'adb_idea'
     updateSinceUntilBuild false
+    intellij.instrumentCode false
 
-    intellij.alternativeIdePath = project.hasProperty("StudioRunPath") ? StudioRunPath : StudioCompilePath
+    intellij.localPath = StudioCompilePath
```

Comment:
I'd rather keep the `project.hasProperty("StudioRunPath") ? StudioRunPath : StudioCompilePath` here.
It allows me to run a different studio than the compile one. Same thing as targetSdk vs minSdk in android. It helps me to test for backward compatibility.